media-translation/cloud-client/translate_from_file.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python
+
+# Copyright 2020 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Google Cloud Media Translation sample application.
+Example usage:
+    python translate_from_file.py resources/audio.raw
+"""
+
+import argparse
+
+# [START media_translation_translate_from_file]
+
+
+def translate_from_file(file_path):
+    from google.cloud import mediatranslation
+
+    client = mediatranslation.SpeechTranslationServiceClient()
+
+    config = mediatranslation.StreamingTranslateSpeechConfig(
+        audio_config=mediatranslation.TranslateSpeechConfig(
```

Comment:
Let's pull this out and create a separate variable for this for readability

Reply:
Done.